### ISSUE TITLE
batch_size

### ISSUE THREAD

**Issue by tyler55427** [River-Zhang/ICEdit#85]
It seems that the batch sizes in paper is 8=1*2*4, but on github is 16=2*2*4?

**Comment by tyler55427** [comment]
In paper:

<img width="696" height="60" alt="Image" src="https://github.com/user-attachments/assets/eec0d9db-9ba2-49e0-8a73-aa4870cb52ae" />

On github:
<img width="659" height="77" alt="Image" src="https://github.com/user-attachments/assets/affa125b-8aca-46aa-a6b6-2211cd121325" />

**Comment by River-Zhang** [comment]
Hi, sorry for the late reply. Thanks for your question! In the original paper, we gave the MoE lora training setting and it is 1*2*4. In the github, we first released the normal lora training setting and it is 2*2*4. Because the moe training requires more gpu memory and can only be trained with a batch size of 1 (when not using gradient checkpointing).

**Comment by tyler55427** [comment]
Get it, thanks!
